projects/loopback.json

```diff
@@ -159,6 +159,62 @@
     "strongloop/strong-error-handler",
     "strongloop/strong-pubsub",
     "strongloop/strong-remoting",
-    "strongloop/strong-soap"
+    "strongloop/strong-soap",
+    "strongloop-community/loopback-connector-redis", 
+    "strongloop-community/loopback-connector-elastic-search", 
+    "strongloop-community/loopback-connector-mysql-jdbc",
+    "strongloop-community/loopback-benchmark-mongodb",
+    "strongloop-community/loopback-acl-route", 
+    "strongloop-community/loopback-connector-sqlite3", 
```

Comment:
I think @cgole needs us to add the milestone markers for accountability/tracking velocity during sprints -- for example I needed to add Sprint 27 milestone for landing that redis fix (of course I prefer less process being agile n all, but it's easier than asking me later what I worked on when she can filter herself via ZH using the milestone label).